lib/app/app.rb

```diff
@@ -100,9 +101,12 @@ class ConnectorsWebApp < Sinatra::Base
   post '/start_sync' do
     job = settings.job_store.create_job
 
+    settings.secret_storage.store_secret(body_params[:content_source_id], { access_token: body_params[:access_token] })
```

Comment:
I think we lack a unified concern to validate request params. What if `content_source_id` or `access_token` is missing? I think it will just return 500 Internal Server Error